View/Helper/FormShimHelper.php

```diff
@@ -43,7 +43,10 @@ public function create($model = null, $options = []) {
 		if (isset($options['action'])) {
 			trigger_error('Using key `action` is deprecated, use `url` directly instead.', E_USER_DEPRECATED);
 		}
-
+		if (isset($options['inputDefaults'])) {
+			$message = "Using key 'inputDefaults' is deprecated.'";
```

Comment:
` better than ' - you also have a trailing extra one at the end by accident here.